src/migrations/20181203153000_move_out_bundles_metadata.js

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+// eslint-disable-next-line import/prefer-default-export
+export const up = async (db, config, logger) => {
+  await db.collection('bundles').ensureIndex({bundleId : 1}, {unique: true});
```

Comment:
In earlier migration we currently have `await db.collection('bundles').ensureIndex({bundleId: 1});`. According to docs, ensureIndex will just ignore the unique param. Also it's deprecated. Let's use createIndex from now on

Reply:
`ensureIndex` is alias for `createIndex`. Switching between these won't change anything.

Reply:
Maybe we should then delete index from previous migration and then create new, with `unique`?

Reply:
It's not. Create index throws error when same index exists with different parameters. And don't always believe mongo documentation :D